src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall(){
+        Color randomColor = ColorSupplier.getRandomColor();
+        Random random = new Random();
+        int value = random.nextInt(100);

```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)

Reply:
move 100 to constant with proper naming